src/main/index.ts

```diff
@@ -4,20 +4,21 @@ import { initAndBind } from '@sentry/core';
 import { defaultIntegrations } from '@sentry/node';
 import { ElectronOptions } from '..';
 import { MainClient } from './client';
-import { Electron } from './integrations/electron';
-import { OnUncaughtException } from './integrations/onuncaughtexception';
+import { Electron, OnUncaughtException, SDKInformation } from './integrations';
 import { NetTransport } from './transports/net';
 export { MainClient } from './client';
 export { MainBackend } from './backend';
+export { Integrations as NodeIntegrations } from '@sentry/node';
+
+// tslint:disable-next-line:variable-name
+export const ElectronIntegrations = { Electron, OnUncaughtException, SDKInformation };
```

Comment:
It's not consistent with what we have in `node` and `browser`. We export `Integrations = [...]` there. So an Array, not object.